users_ldap_mail/users_ldap_model.py

```diff
@@ -19,32 +19,29 @@
 #
 ##############################################################################
 
-from openerp.osv import fields, orm
+from openerp import models, fields
+from openerp import SUPERUSER_ID
 
 import logging
-_log = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)
 
 
-class CompanyLDAP(orm.Model):
+class CompanyLDAP(models.Model):
     _inherit = 'res.company.ldap'
-    _columns = {
-        'name_attribute': fields.char(
-            'Name Attribute', size=64,
-            help="By default 'cn' is used. "
-                 "For ActiveDirectory you might use 'displayName' instead."),
-        'mail_attribute': fields.char(
-            'E-mail attribute', size=64,
-            help="LDAP attribute to use to retrieve em-mail address."),
-    }
 
-    _defaults = {
-        'name_attribute': 'cn',
-        'mail_attribute': 'mail',
-    }
+    name_attribute = fields.Char(
+        'Name Attribute',
+        help=("By default 'cn' is used. or ActiveDirectory you might use "
+              "'displayName' instead."),
+        default='cn')
+    mail_attribute = fields.Char(
+        'Email Attribute',
+        help="LDAP attribute to use to retrieve email address.",
+        default='mail')
 
     def get_ldap_dicts(self, cr, ids=None):
```

Comment:
I tried api.multi before and it gave me an error (expecting more arguments, etc) because it assumes the signature is (cr, uid, ids, *arguments, context), but that is not the signature in res.company.ldap for the function being overridden.